// @novu/shared/dist/cjs/entities/notification-template/notification-template.interface.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.TriggerContextTypeEnum = exports.TriggerTypeEnum = exports.IGroupedBlueprint = exports.NotificationTemplateTypeEnum = void 0;
var NotificationTemplateTypeEnum;
(function (NotificationTemplateTypeEnum) {
    NotificationTemplateTypeEnum["REGULAR"] = "REGULAR";
    NotificationTemplateTypeEnum["ECHO"] = "ECHO";
})(NotificationTemplateTypeEnum = exports.NotificationTemplateTypeEnum || (exports.NotificationTemplateTypeEnum = {}));
class IGroupedBlueprint {
}
exports.IGroupedBlueprint = IGroupedBlueprint;
var TriggerTypeEnum;
(function (TriggerTypeEnum) {
    TriggerTypeEnum["EVENT"] = "event";
})(TriggerTypeEnum = exports.TriggerTypeEnum || (exports.TriggerTypeEnum = {}));
var TriggerContextTypeEnum;
(function (TriggerContextTypeEnum) {
    TriggerContextTypeEnum["TENANT"] = "tenant";
    TriggerContextTypeEnum["ACTOR"] = "actor";
})(TriggerContextTypeEnum = exports.TriggerContextTypeEnum || (exports.TriggerContextTypeEnum = {}));
